### PR TITLE
Add returning workflows.run API response

### PR DESCRIPTION
**Pull request by timidri**
This PR adds the returning of the API response of the workflows.run method.

### CODE REVIEW

File: src/relay_sdk/workflows.py
Comment (impl) [inline src/relay_sdk/workflows.py]: I think it might be nice to have this be represented by an actual "model" type, which is luckily quite easy with Python [dataclasses](https://docs.python.org/3/library/dataclasses.html) when the data isn't nested.

```python
@dataclass
class Run:
    name: str
    run_number: int
    app_url: str

class Workflows:
    def run(...):
        # ...
        return Run(**r.json())
```

Also, we shouldn't ignore E501. You can split the line into multiple lines, so something like this will be fine:

```python
def run(self, name: str,
        parameters: Mapping[str, Any] = {}) -> Run:
```

Comment (timidri) [reply src/relay_sdk/workflows.py]: @impl I've made the changes, thanks for the comments! Waiting for approval.

**Review by impl — APPROVED** [review]
LGTM, don't forget to update the CHANGELOG before you cut the release!

**Pull request merged**
